test/postgresql.migration.test.js

```diff
@@ -73,6 +73,42 @@ describe('migrations', function() {
       done();
     });
   });
+
+  it('OrderData should have correct prop type uuid with custom generation function', function(done) {
```

Comment:
test title should be able to be read like a sentence and also be generic (not specific to `OrderData`) e.g. `it('checks that the correct prop type uuid with custom generation function is used`) or something

Reply:
I use the same format of previous test. Do you want me to change that one as well?

Reply:
It's okay then. I don't think it's important for this PR.